imageroot/systemd/user/php7.4-fpm.service

```
@@ -0,0 +1,24 @@
+[Unit]
+Description=Podman  php7.4-fpm.service
+BindsTo=webserver.service
+After=nginx.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=%S/state/environment
+Restart=always
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/php7.4-fpm.pid %t/php7.4-fpm.ctr-id
+ExecStart=/usr/bin/podman run --conmon-pidfile %t/php7.4-fpm.pid \
+    --cidfile %t/php7.4-fpm.ctr-id --cgroups=no-conmon \
+    --pod-id-file %t/webserver.pod-id --replace -d --name  php7.4-fpm \
+    --env-file=%S/state/environment \
+    -v websites:/app:Z -v %h/.config/php7.4-fpm-custom.d:/opt/bitnami/php/etc/php-fpm.d:Z \
```

Set the unit working directory

    WorkingDirectory=%S/state
 
Like said above, create writable directories under $AGENT_STATE_DIR, then the volume mapping rules become

```suggestion
    --volume=websites:/app:z --volume=./php7.4-fpm-custom.d:/opt/bitnami/php/etc/php-fpm.d:Z \
```

- note that websites must be mounted with :z (shared) flag, as more containers use the same volume
- :bulb: shortening the source dir name, trimming "custom":  `--volume=./php7.4-fpm.d:/opt/bitnami/php/etc/php-fpm.d:Z`

See https://github.com/NethServer/ns8-core/blob/main/ldapproxy/imageroot/systemd/user/ldapproxy.service#L8

---

done